### PR TITLE
Update stylelint.config.js

### PR DESCRIPTION
**Pull request by lovetillyourlastpedalfalls**
⚛👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.

1. Copy the correct template for your contribution
  - 🐛 Are you fixing a bug? Copy the template from <https://bit.ly/atom-bugfix-pr>
  - 📈 Are you improving performance? Copy the template from <https://bit.ly/atom-perf-pr>
  - 📝 Are you updating documentation? Copy the template from <https://bit.ly/atom-docs-pr>
  - 💻 Are you changing functionality? Copy the template from <https://bit.ly/atom-behavior-pr>
2. Replace this text with the contents of the template
3. Fill in all sections of the template
4. Click "Create pull request"

### CODE REVIEW

File: stylelint.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
     'max-empty-lines': null, // TODO: enable?
     'selector-type-no-unknown': null,
     'function-comma-space-after': null, // TODO: enable?
-    'font-family-no-missing-generic-family-keyword': null, // needed for octicons (no sensible fallback)
+    'font-family-no-missing-generic-family-keyword': null, // needed for Actions (no sensible fallback)
```
Comment (lovetillyourlastpedalfalls) [inline stylelint.config.js:11]: 'font-family-no-missing-generic-family-keyword': null, // needed for Actions (no sensible fallback)

**Review by lovetillyourlastpedalfalls — COMMENTED** [review]
'font-family-no-missing-generic-family-keyword': null, // needed for Actions (no sensible fallback)

**Comment by ThatXliner** [comment]
Typo fix?

**Comment by sadick254** [comment]
This PR doesn't fix any typos. The fix on this PR is irrelevant. Am going to close this for now.

**Pull request closed without merging**